README.md

```diff
@@ -1,10 +1,40 @@
-# TUF repository automation demo
+# TUF repository demo
 
 This is a live [TUF](https://theupdateframework.io/) repository, intended for
 testing and demonstrations. It's
 maintained with  [TUF-on-CI](https://github.com/theupdateframework/tuf-on-ci).
 
-TUF clients can access the repository in https://jku.github.io/tuf-demo:
+The goals of this project are:
+* Maintain a long term repository with daily changes that the TUF community can use as a
+  live example and as a remote repository for client testing that requires one
+* Allow anyone in the TUF community to maintain artifacts in the repository for those purposes
+* Act as the interactive testing platform for TUF-on-CI (the repository maintenance tool)
+
+Being a secure delivery mechanism for any specific artifacts is not a goal: this is a demo. If you have questions, the
+[TUF-on-CI slack](https://cloud-native.slack.com/archives/C04SHK2DPK9) is a good place.
+
+### Becoming a signer
+
+We'll try to offer anyone in the TUF community the ability to add and modify artifacts in this repository. 
+
+The process is not formalized yet but if you'd like to become a signer please open an issue and we'll
+go from there:
+* As a general rule you can expect to get your own delegation where you can add artifacts as you want
+  (relatively small ones for now please: the system only stores artifacts in git at the moment)
+* A Yubikey or similar hardware key is required as a signing method (Sigstore may be an option in future)
```

Comment:
the issues with sigstore are related to the non-standard key and signature formats: they're only supported by python-tuf (and even then the client needs to enable sigstore keys specifically):
* A delegated role with sigstore keys should not break any clients that don't try to use that role... but I'm not sure if that's actually true for all clients
* Only clients that explicitly enable sigstore support would be able to download artifacts from that role